orttraining/orttraining/training_api/checkpoint.h

```diff
@@ -4,43 +4,27 @@
 #pragma once
 
 #include "core/platform/path_lib.h"
-#include "core/platform/env.h"
 #include "onnx/defs/tensor_proto_util.h"
```

Comment:
```suggestion

```
this might not needed by minimal build. 

but looks tensorproto is still available in minimal build, so it is not big deal.